monitor/census.go

```diff
@@ -1047,6 +1049,13 @@ func InitCensus(nodeType NodeType, version string) {
 			TagKeys:     baseTags,
 			Aggregation: view.LastValue(),
 		},
+		{
+			Name:        "ai_num_orchs_avail",
+			Measure:     census.mAINumOrchs,
+			Description: "AI Live number of available orchestrators",
```

Comment:
Nit: I'm not sure if I'd call it "available", because they can actually be unreachable or busy.

Reply:
changed into "known"